src/commands/diff.rs

```diff
@@ -432,13 +464,15 @@ async fn diff_impl(
     if pull_request.is_some() && github_commit_message.is_none() {
         let input = dialoguer::Input::<String>::new()
             .with_prompt("Message (leave empty to abort)")
+            .with_initial_text(message_on_prompt.clone())
             .allow_empty(true)
             .interact_text()?;
 
         if input.is_empty() {
             return Err(Error::new("Aborted as per user request".to_string()));
         }
 
+        *message_on_prompt = input.clone();
```

Comment:
you write to `message_on_prompt` here but never seem to read from it afterwards?

Reply:
When we run over multiple commits, then it will be read the next time `diff_impl` is called, i.e. on the next commit.
The idea is that when you update 5 commits, and you type in "rebase" as the message of the first, then you can just press enter for the next 4 commits to use the same message.